### PR TITLE
Fix refetch loop when impersonate

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Fix bug when impersonate is called and it fails

#### What problem is this solving?
Fix refetch loop when impersonate

#### How should this be manually tested?
[workspace](https://felipe--storecomponents.myvtex.com/)

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by salesfelipe** [comment]
[Fix refetch loop when impersonate](https://trello.com/c/s3eW0Q7H/57-fix-refetch-loop-when-impersonate)

File: react/index.js [hunk 1]
```diff
@@ -62,13 +65,16 @@ class TelemarketingContainer extends Component {
     this.setState({ loadingImpersonate: true })
     const variables = { email }
     impersonate({ variables })
-      .then(() => {
-        session.refetch()
+      .then((res) => {
```
Comment (andreldsa) [inline react/index.js]: res? Maybe `response`

File: react/index.js [hunk 2]
```diff
@@ -46,9 +46,12 @@ class TelemarketingContainer extends Component {
 
     depersonify()
       .then(res => {
-        if (res.data.depersonify) {
+        const depersonify = path(['data', 'depersonify'], res)
```
Comment (andreldsa) [inline react/index.js]: res?

Comment (salesfelipe) [reply react/index.js]: 👍

File: react/index.js (hunk 1 above)
Comment (salesfelipe) [reply react/index.js]: 👍

**Pull request merged**